cli/lib/kontena/cli/common.rb

```diff
@@ -14,8 +14,8 @@ module Common
 
       def_delegators :prompt, :ask, :yes?
       def_delegators :config,
-        :current_grid=, :require_current_grid, :current_master,
-        :current_master=, :require_current_master, :require_current_account,
+        :current_grid=, :require_current_grid, :require_current_master_token,
```

Comment:
No, this breaks if something calls `require_current_master_token` or `require_current_grid` before calling something else that hits `Kontena::Cli::Common#current_master` which does `config.current_master=` as a side-effect.

With a `kontena whoami` modified to include `Kontena::Cli::GridOptions` and print out extra stuff, it looks like this:

```
$ bin/kontena whoami
require_current_master_token.access_token=dc158...
require_current_grid=development
Master: development
current_master.token.access_token=dc158...
URL: http://localhost:9292/
Grid: development
User: terom
$ bin/kontena whoami --master terom-test-v13
require_current_master_token.access_token=dc158...
require_current_grid=development
Master: terom-test-v13
current_master.token.access_token=b1b77...
URL: https://X.Y.Z.W
Grid: test
User: ...@...
```

It doesn't make any sense to me for `require_current_grid` to return something different than `current_grid` etc depending on the order in which you call them.